docs/prerequisites.md

```diff
@@ -27,43 +38,100 @@ Additional AWS credentials are needed for the operator to be successfully instal
 
 ### For STS clusters
 
-1. [Extract and prepare the `ccoctl` binary](https://docs.openshift.com/container-platform/4.11/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts)
+#### Option 1. Using `ccoctl`
```

Comment:
Same comment here for a little intro on which Option to choose, and why.

Reply:
An intro is added.